### DISCUSSION TITLE
Le tengo que tener miedo a esta advertencia al intentar hacer commit?

### DISCUSSION THREAD

**Discussion by ramiroLopezA** [INGCOM-UNRN-P1/cursada-2023#39] [Preguntas y respuestas] [answered]
__Git Bash__ y __Github__ desktop me tiran esta advertencia cuando intento darle a commit:

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141680974/989177f9-eeea-4706-a2e6-deacfae4a78d)

Este error no me deja hacer push, solo me da la opcion de hacer __Force Push__

**Comment by martinvilu** [comment, marked as answer]
No es un error (vos mismo lo decís, es una advertencia), y no requiere de hacer `--force`

Es una diferencia en como se crean los archivos de texto en Windows y Linux que git se encarga de transformar para que sea igual en ambos

**Comment by martinvilu** [comment]
Si queres una respuesta, no le des cerrar si no fue contestada.
